templates/default/interfaces.erb

```diff
@@ -35,23 +35,23 @@ iface <%= @device %> inet <%= @type %>
 <% if @mtu -%>
   mtu <%= @mtu %>
 <% end %>
-<% if @pre_up -%>
-  pre-up <%= @pre_up %>
+<% @pre_up.each do |x| -%>
```

Comment:
That's a good means to keep compatibility.  I'll update my branch shortly.

Thanks!